### DISCUSSION TITLE
i am not able to setup in local

### DISCUSSION THREAD

**Discussion by satpil** [ToolJet/ToolJet#6198] [Q&A]
I am following the setup guide to contribute but not able to setup fully only Frontend run successfully but when run backend server it connected to local postgres but when frontend app request api like /api/session and /api/config it throw forbidden error.

    responseTime: 3
    req: {
      "id": 52,
      "method": "GET",
      "url": "/api/session",
      "query": {},
      "params": {
        "0": "/session"
      },
      "headers": {
        "host": "localhost:3000",
        "connection": "keep-alive",
        "sec-ch-ua": "\"Chromium\";v=\"112\", \"Google Chrome\";v=\"112\", \"Not:A-Brand\";v=\"99\"",
        "sec-ch-ua-mobile": "?0",
        "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36",
        "sec-ch-ua-platform": "\"macOS\"",
        "accept": "*/*",
        "origin": "http://localhost:8082",
        "sec-fetch-site": "same-site",
        "sec-fetch-mode": "cors",
        "sec-fetch-dest": "empty",
        "referer": "http://localhost:8082/",
        "accept-encoding": "gzip, deflate, br",
        "accept-language": "en-GB,en-US;q=0.9,en;q=0.8",
        "cookie": "redirectPath=/; username-localhost-8888=\"2|1:0|10:1682859192|23:username-localhost-8888|44:MTllMmRiYTEyMjNlNDQxM2FlMGY5ODM5YjdiMDE4MDc=|bb6ea83541c730c1241fcbbda6f445ca20a5fb87ca3559c435ea11b5e3dfa86f\"; _xsrf=2|d694a260|3c4b701555ab278e32cb8d00eeb65d26|1682859193"
      },
      "remoteAddress": "127.0.0.1",
      "remotePort": 56280
    }
INFO [04/30/2023, 1:25:48 PM UTC] (16859 on satishs-MacBook-Air.local): request completed
    res: {
      "statusCode": 422,
      "headers": {
        "x-powered-by": "Express",
        "access-control-allow-origin": "http://localhost:8082",
        "vary": "Origin, Accept-Encoding",
        "access-control-allow-credentials": "true",
        "content-security-policy": "img-src * data: blob:;script-src maps.googleapis.com storage.googleapis.com apis.google.com accounts.google.com 'self' 'unsafe-inline' 'unsafe-eval' blob: https://unpkg.com/@babel/standalone@7.17.9/babel.min.js https://unpkg.com/react@16.7.0/umd/react.production.min.js https://unpkg.com/react-dom@16.7.0/umd/react-dom.production.min.js cdn.skypack.dev cdn.jsdelivr.net;default-src maps.googleapis.com storage.googleapis.com apis.google.com accounts.google.com *.sentry.io 'self' blob:;connect-src ws://localhost 'self' *;frame-ancestors *;frame-src *;base-uri 'self';block-all-mixed-content;font-src 'self' https: data:;object-src 'none';script-src-attr 'none';style-src 'self' https: 'unsafe-inline'",
        "content-type": "application/json; charset=utf-8",
        "content-length": "129",
        "etag": "W/\"81-ZKRQPv6WaaD/ArXiYzfDeFA5iPU\""
      }
    }
    responseTime: 14
    req: {
      "id": 53,
      "method": "GET",
      "url": "/api/metadata",
      "query": {},
      "params": {
        "0": "/metadata"
      },
      "headers": {
        "host": "localhost:3000",
        "connection": "keep-alive",
        "sec-ch-ua": "\"Chromium\";v=\"112\", \"Google Chrome\";v=\"112\", \"Not:A-Brand\";v=\"99\"",
        "sec-ch-ua-platform": "\"macOS\"",
        "sec-ch-ua-mobile": "?0",
        "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36",
        "content-type": "application/json",
        "accept": "*/*",
        "origin": "http://localhost:8082",
        "sec-fetch-site": "same-site",
        "sec-fetch-mode": "cors",
        "sec-fetch-dest": "empty",
        "referer": "http://localhost:8082/",
        "accept-encoding": "gzip, deflate, br",
        "accept-language": "en-GB,en-US;q=0.9,en;q=0.8"
      },
      "remoteAddress": "127.0.0.1",
      "remotePort": 56278
    }
![Screenshot 2023-04-30 at 7 12 01 PM](https://user-images.githubusercontent.com/59343220/235356104-2b778e99-40d5-463b-b8ba-fc9dfcbfa2b2.png)

**Comment by Sheikh-JamirAlam** [comment]
I am using Ubuntu and I am following this guide to setup my project locally https://docs.tooljet.com/docs/contributing-guide/setup/ubuntu.
While running `npm run --prefix server db:reset` command I am facing this error. Can someone tell me what to do?
![Screenshot from 2023-04-30 21-12-33](https://user-images.githubusercontent.com/48979376/235362535-aa77f66f-3947-4885-b8c9-da100899f9d3.png)
